### ISSUE TITLE
Bom items

### ISSUE THREAD

**Issue by DS1705** [inventree/inventree-python#190]
is there a way to add bom items to an assembly from python?
with a BOM file
or single item

**Comment by T0jan** [comment]
yes it is possible with the `inventree.part.BomItem.create(api, data= {part, sub_part, quantity})` method. You need at least the pk of the part you want to add the bom item to, the pk of the part you want to assign and the quantity.

**Comment by T0jan** [comment]
with a bom file you could either use the web-ui to upload it directly or you have to write yourself a python script which reads in the bom file and then uses this api to upload them.

**Comment by DS1705** [comment]
okay thanks for the information
